kubernetes-repositories.md

```diff
@@ -73,6 +73,34 @@ Create a broader base of repositories than the existing gh/kubernetes/kubernetes
    * All OWNERS must be members of standing as defined by ability to vote in Kubernetes steering committee elections. in the Kubernetes community
    * Repository must be approved by SIG-Architecture
 
+### Removing Repositories
+
+As important as it is to add new repositories, it is equally important to
+prune old repositories that are no longer relevant or useful.
+
+It is in the best interests of everyone involved in the Kubernetes community
+that our various projects and repositories are active and healthy. This
+ensures that repositories are kept up to date with the latest Kubernetes
+wide processes, it ensures a rapid response to potential required fixes
+(e.g. critical security problems) and (most importantly) it ensures that
+contributors and users receive quick feedback on their issues and 
+contributions.
+
+#### Grounds for removal
+A repository may be removed from the project if it is _not_ an associated
```

Comment:
Should we phrase this as "SIG repositories & core repositories may be removed from the project if any of the following criteria are met:"? Both because I think it's a little clearer, and because technically a SIG repo is an associated repo.

Reply:
done.